__init__.py

```diff
@@ -72,24 +128,9 @@ def INPUT_TYPES(s):
         }
     RETURN_TYPES = ("STRING",)
     RETURN_NAMES = ("text_out",)
-    FUNCTION = "promptComposerCustomLists"
+    FUNCTION = "generatePrompt"

```

Comment:
I refactored this so that all custom nodes can use the same function definition as to avoid duplication